src/e_ibmca.c

```diff
@@ -591,6 +680,17 @@ static int ibmca_init(ENGINE *e)
         || !BIND(ibmca_dso, ica_aes_gcm_initialize)
         || !BIND(ibmca_dso, ica_aes_gcm_intermediate)
         || !BIND(ibmca_dso, ica_aes_gcm_last)
+#endif
+#ifndef OPENSSL_NO_EC
+	|| !BIND(ibmca_dso, ica_ec_key_new)
```

Comment:
Hm this is the all-or-nothing way. Didn't Jörg already add something which is more flexible?
eg. like:
static ica_ec_support_available;
if (BIND(xxx) && BIND(yyy) && BIND(zzz)) {
    ica_ec_support_available = 1;
}
So that ibmca is able to run with older libica versions which do not have ec support.

Reply:
its not related to the ecc feature, it has always been like this. will be addressed in a seperate PR.